ocs_ci/ocs/registry.py

```diff
@@ -343,3 +345,36 @@ def check_image_exists_in_registry(image_url):
         return_value = True
         logger.info("Image exists in Registry")
     return return_value
+
+
+def modify_registry_pod_count(count):
+    """
+    Function to modify registry replica count(increase/decrease pod count)
+
+    Args:
+        count (int): registry replica count to be changed to
+
+    Returns:
+        bool: True in case if changes are applied. False otherwise
+
```

Comment:
this function can raise `TimeoutExpiredError`. Please add `Raises` section to the docstring

Reply:
done